### PR TITLE
Loading animation

### PR DESCRIPTION
**Pull request by kozr**
## :construction_worker: Changes

A brief summary of what changes were introduced.

Breathing loading animation for nwhacks website

## :thought_balloon: Notes

Any additional things to take into consideration.
Loop is 5 seconds total. It will take 2.5s to glow fully and 2.5s to dim.

At first, I was like wtf they gave me two images and told me to make it loop with no animation. BUT then I found this hack where I just have two images and make one dim. LOL

## :flashlight: Testing Instructions

Explain how to test your changes, if applicable.
Check out amplify.
If you wish to see it for longer, then hit pause in sources while loading (with chrome dev tool).
Another way is to increase timeout in index.vue.

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -14,7 +14,7 @@ const AsyncComponent = () => ({
   component: new Promise((resolve, reject) => {
     setTimeout(() => {
       resolve(Main)
-    }, 1000)
+    }, 100000)
```
Comment (ItsAdel) [inline pages/index.vue]: Would this stay at 100 seconds or is that just for demo pusposes? Also what do you think an appropriate length would be considering 5 seconds for full animation loop?

**Review by ItsAdel — APPROVED** [review]
Just one comment about the length of animation but otherwise the animation looks really good!

**Review by rmehri01 — APPROVED** [review]
Like Adel said, I think the timeout needs to change but besides that looks great!

File: pages/index.vue (hunk 1 above)
Comment (kozr) [reply pages/index.vue]: Totally for demo purposes, yeah :)

To be honest, I'm not sure what the length should be. If it's too short, we won't notice the animation at all. If it's too long, then people wait longer. I'll test it out.

**Comment by kozr** [comment]
Flipped opacity so it's more obvious

**Pull request merged**